q2lint.py

```diff
@@ -38,6 +38,9 @@ def main():
                 errors.append("Package dependencies should be stored in a "
                               "conda recipe instead of setup.py "
                               "`install_requires`.")
+            if ('license=\'BSD-3-Clause\',' not in text and
```

Comment:
For both of these string equality checks, can you remove the requirement that there's a trailing comma? That way a `license` kwarg can be the final parameter supplied to `setup()` (devs may omit a trailing comma in that case). This equality check will also match how `install_requires` is linted above.